openpype/hosts/maya/api/plugin.py

```diff
@@ -263,8 +273,14 @@ def update(self, container, representation):
                 "{}:*".format(namespace), type="AlembicNode"
             )
             if alembic_nodes:
-                for attr, value in alembic_data.items():
-                    cmds.setAttr("{}.{}".format(alembic_nodes[0], attr), value)
+                for attr, data in alembic_data.items():
+                    node_attr = "{}.{}".format(alembic_nodes[0], attr)
+                    if data["connected"]:
+                        cmds.connectAttr(
+                            data["attribute"], node_attr, force=True
+                        )
+                    else:
+                        cmds.setAttr(node_attr, data["value"])
```

Comment:
Doesn't updating the alembic automatically reconnect the `time` attribute and thus for us to set the value we'll first need to force disconnect it?

To test:
- Load alembic
- Disconnect time
- Set manually
- Update alembic with this PR

Does that work as expected?

Reply:
Also, this would mean if you'd update without any changes to `.time` attribute it'd probably log a warning or error on update that `time` was already connected?

Reply:
You are right. Will have a look at this.